rest_flex_fields/filter_backends.py

```diff
@@ -109,38 +124,39 @@ def get_schema_fields(self, view):
         if not issubclass(serializer_class, FlexFieldsSerializerMixin):
             return []
 
+        fields = self._get_fields(serializer_class)
         expandable_fields = self._get_expandable_fields(serializer_class)
 
         return [
             coreapi.Field(
-                name="fields",
+                name=FIELDS_PARAM,
                 required=False,
                 location="query",
                 schema=coreschema.String(
                     title="Selected fields",
-                    description="Specify required field by comma",
+                    description="Specify required fields by comma",
                 ),
-                example="field1,field2,nested.field",
+                example=(fields or "field1,field2,nested.field") + "," + WILDCARD_VALUES_JOINED,
```

Comment:
If `fields` is somehow empty then we fall back to the default example string.

In addition, we append the wildcard values to demonstrate their use too.